File: support/src/main/java/com/radioteria/support/services/mail/LoggingEmailService.java
package com.radioteria.support.services.mail;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class LoggingEmailService implements EmailService {
    final private static Logger LOGGER = LoggerFactory.getLogger(LoggingEmailService.class);

    @Override
    public void send(String to, String subject, String body) {
        LOGGER.info("==== LETTER ====");
        LOGGER.info("To:      " + to);
        LOGGER.info("Subject: " + subject);
        LOGGER.info("Body:    " + body);
        LOGGER.info("================");
    }
}

File: support/src/main/kotlin/com/radioteria/support/services/mail/LoggingEmailService.kt
package com.radioteria.support.services.mail

import org.slf4j.LoggerFactory

class LoggingEmailService : EmailService {

    override fun send(to: String, subject: String, body: String) {
        LOGGER.info("==== LETTER ====")
        LOGGER.info("To:      " + to)
        LOGGER.info("Subject: " + subject)
        LOGGER.info("Body:    " + body)
        LOGGER.info("================")
    }

    companion object {
        private val LOGGER = LoggerFactory.getLogger(LoggingEmailService::class.java)
    }

}
